.github/workflows/release.yml

```diff
@@ -74,3 +74,8 @@ jobs:
           artifact-name: io-helidon-build-tools-artifacts-${{ github.ref_name }}
           artifact-path: target/nexus-staging/
           run: etc/scripts/release.sh release_build
+  release-cli:
+    needs: release
+    uses: ./.github/workflows/release-cli.yml
```

Comment:
This will run on every release right ?
If you do it this way you need to filter on the branch name.

Otherwise, you need to call release from release-cli. Although this will mean to define a trigger for `release-cli-*` and exclude that in the release workflow to avoid double builds.

Reply:
Right, I forgot the condition when I rebased my changes from my testing branch.